docs/src/userguide/io/index.rst

```diff
@@ -88,8 +88,66 @@ object, and it will be the responsibility of your own code to setup the input.
    * any automatic data input existence checks (described above)
    * any automatic data output existence checks (described below)
 
+.. _multiple-event-inputs:
+
+Multiple Event Inputs
+^^^^^^^^^^^^^^^^^^^^^
+
+Currently, Klio supports working with multiple event inputs.
+This may be multiple Pub/Sub subscriptions, or BigQuery tables, or any other of Klio's :ref:`supported event inputs <supported-event-io>`.
+A job may not have multiple inputs that support different modes.
+For instance, Pub/Sub is only supported in streaming mode.
+Therefore, a streaming job may not mix reading from a Pub/Sub subscription with any inputs supported in batch mode.
+
+Step 1: Defining Multiple Inputs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To define multiple event inputs, just add another list item under ``job_config.events.inputs`` in the ``klio-job.yaml`` file:
+
+.. code-block:: yaml
+
+  job_config:
+    events:
+      inputs:
+        - type: file
+          location: ./batch_track_ids_1.txt
+        - type: file
+          location: ./batch_track_ids_2.txt
+    # <-- snip -->
+
+Klio will automatically read from the configured inputs.
```

Comment:
Does it read them in order or simultaneously?

Reply:
oh looks like this is covered below!

Reply:
It is probably a good idea to clarify it up here even if it's covered below, just to be super clear for the reader. Good catch!